compiler/src/dmd/backend/x86/code_x86.d

```diff
@@ -74,7 +74,7 @@ enum NUMGENREGS = 16;
 // currently only used as a replacement for mES in cgcod.c
 enum NUMREGS = 25;
 
-enum PSW     = 25;
+enum PSW     = 30;
```

Comment:
`core.exception.ArrayIndexError@src/dmd/backend/x86/cgreg.d(194): index [30] is out of bounds for array of length 29`

Reply:
Yes, that's exactly the problems I want to flush out!